framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/Instance.java

```diff
@@ -15,8 +15,31 @@
 
 package org.kie.cloud.api.deployment;
 
+/**
+ * Cloud instance representation. One instance of deployed application.
```

Comment:
"Cloud instance" should probably be "running instance of deployment" if I understand the api right.